features/move-before.yml

```diff
@@ -1,8 +1,11 @@
 name: moveBefore()
 description: The `moveBefore()` DOM method relocates a node while preserving its state. For example, you can move the active element without losing focus, move an animated element without resetting the animation, or move an iframe without reloading its content.
-spec: https://github.com/whatwg/dom/pull/1307
```

Comment:
Would you mind removing the exception for this from [`scripts/spec.ts`](https://github.com/ddbeck/web-features/blob/efc2321eb4b1d3f4f9c102739fac551653129cd8/scripts/specs.ts#L124-L127)?